source/particle/property/viscoplastic_strain_invariants.cc

```diff
@@ -0,0 +1,195 @@
+/*
+  Copyright (C) 2015 - 2017 by the authors of the ASPECT code.
+
+ This file is part of ASPECT.
+
+ ASPECT is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; either version 2, or (at your option)
+ any later version.
+
+ ASPECT is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with ASPECT; see the file LICENSE.  If not see
+ <http://www.gnu.org/licenses/>.
+ */
+
+#include <aspect/particle/property/viscoplastic_strain_invariants.h>
+#include <aspect/material_model/visco_plastic.h>
+#include <aspect/initial_composition/interface.h>
```

Comment:
Do you need this last include? I think you can remove it.

Reply:
The particles are currently initialized with the initial composition. Is there a way to do that without the include or is it needed?

Reply:
Oh you are right, I did not see that. Keep it as is.